packages/components/src/index.ts

```diff
@@ -30,6 +31,7 @@ export * from './components/RotateDeviceImage/RotateDeviceImage';
 export * from './components/DataAnalytics';
 export * from './components/Divider/Divider';
 export * from './components/Dropdown/Dropdown';
+export * from './components/Dropdown/Menu';
```

Comment:
why do you need this?

Reply:
I'm using GroupedMenuItems and I used to import it like this:

` import { GroupedMenuItems } from '@trezor/components/src/components/Dropdown/Menu';`

Think it's better to import it from `@trezor/components`

Reply:
I will be then specific and change it to:

```js
export type { GroupedMenuItems } from './components/Dropdown/Menu';
```